mlflow-tests/tests/base.py

```
@@ -0,0 +1,429 @@
+        In K8s mode, uses token-based authentication (Bearer token).
+    """
+    username = user_info[0]
+    credential = user_info[1]
+
+    logger.info("=" * 80)
+    logger.info("SETTING USER AUTHENTICATION CONTEXT")
+    logger.info("=" * 80)
+
+    if Config.LOCAL:
+        # LOCAL mode: Basic authentication with username/password
+        logger.info(f"Authentication mode: LOCAL (Basic Auth)")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)